### DISCUSSION TITLE
Releases missing

### DISCUSSION THREAD

**Discussion by Timple** [Arnold-n/P1P2MQTT#39] [General]
I notice a lot of development in this repo. Awesome!

Is there a reason not all releases make it to the releases page? https://github.com/Arnold-n/P1P2Serial/releases
I would prefer not to compile them myself as to exclude me (and my pc) from messing up the build 🙂

**Comment by Arnold-n** [comment]
As the release generation is not yet automated, I tend to do it only for major steps in development. Like the planned HA integration. But to increase release frequency as you are interested in I will also look into automation.

**Comment by Timple** [comment]
Ah, so it was intentional.

If there is not much demand for up-to-date releases, there is no need to bother :slightly_smiling_face:

**Comment by stomko11** [comment]
Is there any ETA on HA integration release?

**Comment by Arnold-n** [comment]
No ETA, as I struggle to find time currently, but should be "soon", this or next month.
